GOVERNANCE.md

```diff
@@ -0,0 +1,158 @@
+
+## Values
+
+The Prometheus developers and community are expected to follow the values defined in the [CNCF charter][charter], including the [CNCF Code of Conduct][coc]. Furthermore, the Prometheus community strives for kindness, giving feedback effectively and building a welcoming environment. The Prometheus developer community strives to operate by rough consensus for the majority of decisions and only resort to conflict resolution through a motion if consensus cannot be reached.
+
+## Projects
+
+Each project must have a MAINTAINERS.md file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced on the prometheus-users mailing list. Any new repositories should be first proposed on prometheus-team@ and following the voting procedures listed below. When a repository is no longer relevant, it should be moved to the prometheus-junkyard Github organization.
+
+## Decisionmaking
+
+### Team members 
+
+Team member status is given to those who have made ongoing contributions for at least 3 months to the Prometheus Project. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account. 
+
+New members may be proposed by any existing member by email to [prometheus-team][team] and voted on by [consensus](#consensus). If no consensus is reached, the new member is not accepted.
+
+Team members are added to the [GitHub organization][gh] and have commit rights to all projects. They should however respect the maintainers of each project.
+
+Team members may retire at any time by emailing [the team][team].
+
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+
+Upon death of a member, their team membership ends automatically.
+
+Unless voted upon otherwise, membership changes take effect immediately. Unless voted upon otherwise, any membership change is announced to [the public developer mailing list][devs] as soon as it is in effect.
+
+### Maintainers
+
+Maintainers are team members who are lead one or more project(s) and serve as a point of conflict resolution amongst the contributors to this project.
+
+A maintainer or committer may resign by notifying the [team mailing list][team]. A maintainer with no project activity for a year is considered to have resigned. Maintainers who wish to resign are encouraged to propose another team member to take over the project.
+
+An extraordinary change of maintainership of any project may be proposed on the [team mailing list][team] and is voted on by [consensus](#consensus). If no consensus can be reached, the matter is resolved by [majority vote](#majority-vote).
```

Comment:
since we moved away from maintainers as the voting base, we didn't have a very clear procedure for involuntary maintainership changes. I tried to stay close to the membership process, but with a lower voting bar.

Reply:
I'd skip the consensus step on this one. If you can override it anyways...

Reply:
ack